src/Version_6_1/SendReturn/SendReturnInitiator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+using CommonMessages;
+using NServiceBus;
+
+public static class SendReturnInitiator
+{
+
+    public static void InitiateSendReturn(this IEndpointInstance bus)
+    {
+        foreach (var endpoint in EndpointNames.All)
```

Comment:
Should we make this async as well to align with other of those extensions? Does it need `Task.Run`?